lib/Common/Memory/Recycler.h

```diff
@@ -784,8 +787,9 @@ class Recycler
     DListBase<GuestArenaAllocator> guestArenaList;
     DListBase<ArenaData*> externalGuestArenaList;    // guest arenas are scanned for roots
 
-#ifdef RECYCLER_PAGE_HEAP
```

Comment:
Why was this moved?

Reply:
I believe I hit a build error at some point in some config. I will undo this change. It seems alright now.

---
In reply to: [148169931](https://github.com/Microsoft/ChakraCore/pull/4104#discussion_r148169931) [](ancestors = 148169931)